sdk/core/azure-core/test/ut/base64_test.cpp

```diff
@@ -125,3 +125,41 @@ TEST(Base64, Roundtrip)
     EXPECT_EQ(Convert::Base64Decode(Convert::Base64Encode(data)), data);
   }
 }
+
+TEST(Base64, InvalidDecode)
+{
+  EXPECT_NO_THROW(Convert::Base64Decode(Convert::Base64Encode(std::vector<uint8_t>{})));
+  EXPECT_NO_THROW(Convert::Base64Decode(""));
```

Comment:
Can we move out the "valid", no_throw test cases to another named test (maybe something like BasicValidDecode), and out of the `InvalidDecode` test. This test should only contain the EXPECT_THROW test cases.